sql/amo_prod/amo_stats_dau_v2/metadata.yaml

```
@@ -0,0 +1,15 @@
+friendly_name: AMO Stats DAU
+description: >
```

Nit: To prevent trialing newlines in case this description shows up in BigQuery
```suggestion
description: >-
```